ts/receiver/queuedJob.ts

```diff
@@ -276,6 +276,8 @@ async function handleRegularMessage(
     conversation.set({
       active_at: message.get('sent_at'),
       lastMessage: message.getNotificationText(),
+      lastMessageInteractionType: message.get('interactionNotification')?.interactionType,
+      lastMessageInteractionStatus: message.get('interactionNotification')?.interactionStatus,
```

Comment:
We have it because if we failed to leave a convo/group we have an error message in the ConversationListItem but if we send or receive a message the lastMessage must be updated to that new message.

Essentially this point of yours.
> it is to reset the lastMessageInteractionType state of the conversation when receiving a message in that conversation?

Reply:
ok all good then